packages/cli/src/databases/migrations/mysqldb/1669739707125-AddWorkflowHashColumn.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import config from '@/config';
+import { v4 as uuidv4 } from 'uuid';
+
+export class AddWorkflowHashColumn1669739707125 implements MigrationInterface {
+	name = 'AddWorkflowHashColumn1669739707125';
+
+	async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+
+		const tablePrefix = config.getEnv('database.tablePrefix');
+
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_entity ADD COLUMN hash CHAR(36)`,
+		);
+
+		const workflowIds: Array<{ id: number }> = await queryRunner.query(`
+			SELECT id
+			FROM ${tablePrefix}workflow_entity
+		`);
+
+		workflowIds.map(({ id }) => {
+			const [updateQuery, updateParams] = queryRunner.connection.driver.escapeQueryWithParameters(
```

Comment:
Some UUIDs might need escaping? Or why do we need this?

Reply:
No special reason, it's just good practice to send queries through the escape mechanism

Reply:
doesn't the query-runner do that automatically in the next line?